libkbfs/folder_branch_ops.go

```diff
@@ -2267,7 +2255,13 @@ func (fbo *folderBranchOps) createEntryLocked(
 	// Passthrough journal writes temporarily.
 	if excl == WithExcl {
 		if jServer, err := GetJournalServer(fbo.config); err == nil {
-			wasEnabled, err := jServer.Disable(ctx, fbo.id())
```

Comment:
Yeah, that might work, I was thinking along the same lines in my last sentence above. Ok, I'll try that tomorrow, thanks